src/tests/gfx_window.edn

```diff
@@ -0,0 +1,11 @@
+(def timestep (/ 1.0 120.0))
+(def Root (Node))
+(def test-chain
+  (Chain
+   "test-chain"
+   :Looped
+   (GFX.MainWindow
+    :Title "SDL Window" :Width 1280 :Height 720 :Debug false)))
+
+(schedule Root test-chain)
+(run Root timestep)
```

Comment:
Nit: missing newline at the end of the file.

Reply:
Fixed, and all other occurences